common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -527,12 +541,14 @@ class @Problem
         answered = false
       if bind
         $(select_field).on 'change', (e) =>
-          @submitAnswersAndSubmitButton()
+          @submitAnswersAndSubmitButton false, true
           return
         return
 
     if answered
       @enableSubmitButton true
+      if data_changed
+        @saveNotification.remove()
```

Comment:
But if you look at the usages of the method, "true" is always passed, except in the case of initialization (when it doesn't harm to call the method anyway).